### PR TITLE
Remove Status field from FrontendConfig

### PR DESCRIPTION
**Pull request by spencerhance**
Regen FrontendConfig and update hack/update-codegen.sh script

### CODE REVIEW

File: pkg/apis/frontendconfig/v1beta1/types.go [hunk 1]
```diff
@@ -28,9 +28,7 @@ import (
 type FrontendConfig struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-
 	Spec   FrontendConfigSpec   `json:"spec"`
```
Comment (bowei) [inline pkg/apis/frontendconfig/v1beta1/types.go]: don't need Spec

Comment (bowei) [reply pkg/apis/frontendconfig/v1beta1/types.go]: Hmm unfortunately BackendConfig already have spec and status. Let's keep it consistent w/ that, sorry about the churn.

Comment (spencerhance) [reply pkg/apis/frontendconfig/v1beta1/types.go]: I see your point, but i'm worried that differing from BackendConfig will be confusing for users

Comment (spencerhance) [reply pkg/apis/frontendconfig/v1beta1/types.go]: okay no worries, I'll close this

**Pull request closed without merging**